testcases/OpTestPrdDriver.py

```diff
@@ -186,7 +186,7 @@ def runTest(self):
             self.skipTest("OpenPower specific")
         # In P9 FSP systems we need to enable this test
         self.prd_init()
-        l_con = self.cv_SYSTEM.sys_get_ipmi_console()
+        l_con = self.cv_SYSTEM.cv_HOST.get_ssh_connection()
```

Comment:
And also for PRD tests we need to use serial console instead of SSH connection, there may system/os crashes/lockups etc we need to capture through console. SSH may terminate.

Reply:
Fixed